.github/CODEOWNERS

```diff
@@ -2,6 +2,6 @@
 # https://help.github.com/en/articles/about-code-owners
 
 # Restricted Paths
-*                                        @TooTallNate @EndangeredMassa @cb1kenobi @Ethan-Arrowood @trek
+*                                        @TooTallNate @EndangeredMassa @Ethan-Arrowood @trek
```

Comment:
@Ethan-Arrowood shall we leave you on here for now?

Reply:
Sure. I don't think I'll be contributing to it much, but would be useful for code review purposes